.github/workflows/benchmark.yml

```diff
@@ -0,0 +1,173 @@
+        default: 'zeebe-cluster'
+        type: string
+        required: false
+      cluster-region:
+        description: 'Specifies the cluster region. Needed to retrieve cluster credentials'
+        default: europe-west1-b
+        type: string
+        required: false
+      load:
+        description: 'Specifies which benchmark components to deploy. `starter`, `timer` and `publisher` can be assigned with the rate at which they publish'
+        default: 'worker, starter=200'
+        type: string
+        required: false
+jobs:
+  build-zeebe-image:
+    name: Build zeebe image
+    runs-on: ubuntu-latest
+    timeout-minutes: 30
+    steps:
+      - uses: actions/checkout@v3
+        with:
+          ref: ${{ inputs.branch }}
+      - uses: google-github-actions/auth@v1
+        with:
+          credentials_json: '${{ secrets.GOOGLE_CREDENTIALS }}'
+      - uses: google-github-actions/setup-gcloud@v1
+        with:
+          project_id: zeebe-io
+          install_components: gke-gcloud-auth-plugin, kubectl
+      - run: gcloud auth configure-docker
+      - uses: ./.github/actions/setup-zeebe
+        with:
+          maven-cache: 'true'
+      - uses: ./.github/actions/build-zeebe
+        id: build-zeebe
+        with:
+          maven-extra-args: -T1C
+      - uses: ./.github/actions/build-docker
+        with:
+          repository: gcr.io/zeebe-io/zeebe
+          version: ${{ inputs.name }}
+          push: true
+          distball: ${{ steps.build-zeebe.outputs.distball }}
+  deploy-benchmark-cluster:
+    name: Deploy benchmark cluster
+    needs:
+      - build-zeebe-image
+    runs-on: ubuntu-latest
+    timeout-minutes: 30
+    steps:
+      - uses: actions/checkout@v3
+        with:
+          ref: ${{ inputs.ref }}
+      - uses: google-github-actions/auth@v1
+        with:
+          credentials_json: '${{ secrets.GOOGLE_CREDENTIALS }}'
+      - uses: google-github-actions/setup-gcloud@v1
+        with:
+          project_id: zeebe-io
+          install_components: gke-gcloud-auth-plugin, kubectl
+      - uses: google-github-actions/get-gke-credentials@v0.8.2
+        with:
+          cluster_name: ${{ inputs.cluster }}
+          location: ${{ inputs.cluster-region }}
+      - uses: azure/setup-helm@v3
+      - name: Add camunda helm repo
+        run: |
+          helm repo add camunda https://helm.camunda.io
+          helm repo update
+      - name: Deploy benchmark installation
+        run: >
+          helm upgrade --install
+          --namespace ${{ inputs.ref }}
+          --create-namespace
+          ${{ inputs.name }} camunda/camunda-platform
+          -f benchmarks/setup/default/zeebe-values.yaml
+          --set zeebe.image.repository=gcr.io/zeebe-io/zeebe
+          --set zeebe.image.tag=${{ inputs.name }}
+          --set zeebe-gateway.image.repository=gcr.io/zeebe-io/zeebe
+          --set zeebe-gateway.image.tag=${{ inputs.name }}
+  deploy-benchmark-load:
+    name: Deploy benchmark load
```

Comment:
:thought_balloon: Asking myself, whether we can't merge both? Here we need to authorize again to adjust the benchmark, not sure whether this makes sense.

Reply:
Yeah we could :+1: It's a tradeoff between granular jobs that allow retrying only the parts of a workflow that failed and larger jobs that save some overhead. Here I agree with you that it makes sense to merge them :+1: